ios/FluentUI/Controls/IndeterminateProgressBarView.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+@available(*, deprecated, renamed: "IndeterminateProgressBarView")
+public typealias MSIndeterminateProgressBarView = IndeterminateProgressBarView
+
+/**
+Indeterminate progress bar view
+ */
+@objc(MSFIndeterminateProgressBarView)
+open class IndeterminateProgressBarView: UIView {
+	
+	/// progressbar's track
+	var trackLayer = CALayer()
+
+	/// gradient layer that slides across the track
+	var gradientLayer = CAGradientLayer()
+	
+	@objc override public init(frame: CGRect) {
+		super.init(frame: frame)
+	}
+	
+	required public init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+	
+	open override func layoutSubviews() {
+		super.layoutSubviews()
+		
+		trackLayer.frame = CGRect(x: 0, y: 0, width: frame.width, height: frame.height)
+		layer.addSublayer(trackLayer)
+	
+		addGradientLayer()
+		addAnimation()
+	}
+	
+	open override func didMoveToWindow() {
+		super.didMoveToWindow()
+		
+		if let window = window {
+			let lightColor : CGColor = Colors.primary(for: window).withAlphaComponent(0).cgColor
+			let darkColor : CGColor = Colors.primary(for: window).cgColor
+			
+			gradientLayer.colors = [lightColor, darkColor, lightColor]
```

Comment:
>gradientLayer [](start = 3, length = 13)

should clients decide what their color should be? or is it by design that it will always be app brand color?

Reply:
It should always be App brand color.